share/new_eds/testing.go

```diff
@@ -0,0 +1,276 @@
+package eds
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/nmt"
+	"github.com/celestiaorg/rsmt2d"
+
+	"github.com/celestiaorg/celestia-node/share"
+	"github.com/celestiaorg/celestia-node/share/eds/edstest"
+	"github.com/celestiaorg/celestia-node/share/sharetest"
+	"github.com/celestiaorg/celestia-node/share/shwap"
+)
+
+type createAccessor func(eds *rsmt2d.ExtendedDataSquare) Accessor
+
+func TestAccessorSample(
```

Comment:
Do we need these testing funcs exported?

Reply:
Yes, they can be used for any Accessor implementation externaly